index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Technical Documentation</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <nav id="navbar">
+    <header>Something about Sofware development</header>
```

Comment:
I recommend using the <h1> tag for the title, remember that <header> element represents a container for introductory content or a set of navigational links.

Reply:
> I recommend using the `<h1>` tag for the title. Remember that `<header>` element represents a container for introductory content or a set of navigational links.

- [ ] I would kindly suggest you to phrased the comments politely ,a respectful words not to use a command statement ,a respectful words like "I would kindly suggest you to use `<h1>` tag for the title for a better semantic reason .
- [ ] very good by providing a reason why you suggest the developer to use `<h1>`tag